### ISSUE TITLE
Keyboad Shortcuts missing checkbox instructions

### ISSUE THREAD

**Issue by terracoda** [phetsims/molarity#233]
With several issues about Keyboard Shortcut dialogs on the go https://github.com/phetsims/joist/issues/668#issuecomment-704600788, https://github.com/phetsims/scenery-phet/issues/630, https://github.com/phetsims/faradays-law/issues/176, https://github.com/phetsims/ratio-and-proportion/issues/82, I just happen to notice that Molarity's Keyboard dialog is missing "checkbox" instructions

@zepumph, can you add it in?

![Screen Shot 2020-10-09 at 1 51 00 PM](https://user-images.githubusercontent.com/1902116/95609317-149a4400-0a39-11eb-91d2-f02f974a2c04.png)

**Comment by terracoda** [comment]
Apologies for missing this when we published.

**Comment by zepumph** [comment]
Excellent! I just saw this issue, but fixed this this morning over in https://github.com/phetsims/molarity/commit/fa5477cf226d11aa96095215b7c97e1889997ebb via https://github.com/phetsims/joist/issues/669. Closing
